### PR TITLE
fix: modify the verification method of es address and port

### PR DESCRIPTION
**Pull request by UPYOI**
#416 
fix the port cannot be added when verifying the port

### CODE REVIEW

**Review by wtt40122 — DISMISSED** [review]
+LGTM

**Review by wtt40122 — DISMISSED** [review]

File: ozhera-log/log-manager/src/test/java/com/xiaomi/mone/log/manager/service/impl/MilogMiddlewareConfigServiceImplTest.java
Comment (sadadw1) [inline ozhera-log/log-manager/src/test/java/com/xiaomi/mone/log/manager/service/impl/MilogMiddlewareConfigServiceImplTest.java:1]: pls delete the XiaoMi's domain address, eg: zjynewretail.api.es.srv

Comment (UPYOI) [reply ozhera-log/log-manager/src/test/java/com/xiaomi/mone/log/manager/service/impl/MilogMiddlewareConfigServiceImplTest.java:1]: done

**Review by sadadw1 — APPROVED** [review]
+LGTM

**Review by wtt40122 — APPROVED** [review]

**Pull request merged**